changelog.md

```diff
@@ -1,4 +1,28 @@
- <a name="6.4.0"></a>
+<a name="6.4.1"></a>
+# 6.4.1 (2020-10-02)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.4.0...v6.4.1)
+
+This is a minor release to fix an issue in the npm package where v6.4.0 was published as ~15GB.
+
+## New Contributors
+
+Thanks to our new contributor 👽🐷🐰🐯🐻! 
+
+- Csaba Palfi @csabapalfi
+
+## Core
+
+* prevent attribute truncation side-effects ([#11503](https://github.com/GoogleChrome/lighthouse/pull/11503))
+
+## Tests
+
+* run every smoke test (except forms) for bundle ([#11493](https://github.com/GoogleChrome/lighthouse/pull/11493))
+
+## Misc
+
+* add chromium webtests to npmignore ([#11512](https://github.com/GoogleChrome/lighthouse/pull/11512))
```

Comment:
really the important part for this release was the `.tmp` part of this PR :)

Reply:
well the only things inside `.tmp` was chromium web test stuff. this message seems to get at the heart of it already, no?

Reply:
ok, fair :)